pkg/build/builder/docker.go

```diff
@@ -210,6 +214,11 @@ func (d *DockerBuilder) addBuildParameters(dir string) error {
 	instructions := dockerfile.ParseTreeToDockerfile(node)
 
 	// Overwrite the Dockerfile.
+	f, err := os.Open(dockerfilePath)
```

Comment:
The ioutil.WriteFile function opens the file for write and closes it. There's no need to open it beforehand
(https://golang.org/pkg/io/ioutil/#WriteFile)